overlord/devicestate/handlers_install.go

```diff
@@ -141,9 +143,15 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
-// TODO:UC20: set to real TPM availability check function
 var checkTPMAvailability = func() error {
```

Comment:
this actually looks like the thing we want to put in a package and have dummy versions of

Reply:
This is used to enable/disable encryption but this could potentially use something that's not TPM, maybe we could use a more generic name and let the exact technology be determined by the package?

Reply:
I don't think is the right time to try to find those names. My main concern is that that package should try not to leak secboot/go-tpm types to make our life seasier.  About that package secboot is probably ok. It's a goal that we never have to import it and the other secboot in tests, except its own tests.